docs/docs/reference/release-notes/v4.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: "2022-01-25"
+version: "4.6.0"
+title: "v4.6 Release Notes"
+---
+
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
+
+## Reuse cache across instances
+
+Walking nodes to track inline objects is very expensive. Caching this across instances of the graphql query sped up subsequent queries by ~10-15% (depending on complexity of nodes). Every performance improvement is important to use as we constantly look for way to make Gatsby faster than ever 🚀
+
+## Improve live editing experience for images in markdown files
+
+Images seen in markdown are not actually registered as node dependencies for the given page, this means that images won't get updated on the fly if the image content changes and the image name doesn't change. In this release, we addressed this issues and we believe this will lead to a better developer expreience for all our users
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v4.5)
+
+[Full changelog][full-changelog]
+
+---
+
+## Notable Bugfixes & Improvements
+
+- `gatsby-transform-remark` and `gatsby-remark-images`: Fixed caching issue where `gatsby-transformer-remark` would not reload image dependencies from `gatsby-remark-images` while running `gatsby develop`, via [PR #34433](https://github.com/gatsbyjs/gatsby/pull/34433)
+- `gatsby-plugin-utils` fixes to plugin schema validation via [PR #34182](https://github.com/gatsbyjs/gatsby/pull/34182):
+  - `pluginOptionsSchema` will give warnings instead of throw errors when a plugin receives an uknown configuration key
+  - `pluginOptionsSchema` will respect default configuration keys set with Joi
+  - `testPluginOptionsSchema` function will also return `hasWarnings` and `warnings` properties for use in test suites
```

Comment:
This is a breaking change, so it should be its own heading and a key highlight of the release